klarna-on-demand/src/test/java/com/klarna/ondemand/UrlHelperTest.java

```diff
@@ -0,0 +1,82 @@
+package com.klarna.ondemand;
+
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mockito;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PowerMockIgnore;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.rule.PowerMockRule;
+import org.robolectric.RobolectricTestRunner;
+import org.robolectric.annotation.Config;
+
+import java.util.Locale;
+
+
+@RunWith(RobolectricTestRunner.class)
+@Config(emulateSdk = 18)
+@PrepareForTest(Context.class)
+@PowerMockIgnore({ "org.mockito.*", "org.robolectric.*", "android.*" })
+public class UrlHelperTest {
+
+    private String token = "my_token";
+
+    @Rule
+    public PowerMockRule rule = new PowerMockRule();
+
+    @Before
+    public void beforeEach() {
+        PowerMockito.mockStatic(Context.class);
+        Mockito.when(Context.getApiKey()).thenReturn("test_skadoo");
+    }
+
+    //region .registrationUrl
+    @Test
+    public void registrationUrl_ShouldReturnPlaygroundUrl_WhenTokenIsForPlayground() {
+        Assert.assertTrue(UrlHelper.registrationUrl().startsWith("https://inapp.playground.klarna.com/registration/new"));
+    }
+
+    @Test
+    public void registrationUrl_ShouldReturnProductionUrl_WhenTokenIsForProduction() {
+        PowerMockito.mockStatic(Context.class);
```

Comment:
When you mock something that's already mocked, it replaces the return values. 
Yes, mocks clear after each test (both static and non-static mocks).
I've also revalidated this.